igor/igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/HelmOciDockerRegistryCache.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2025 Harness, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.igor.docker;
+
+import static java.lang.String.format;
+
+import com.netflix.spinnaker.igor.IgorConfigurationProperties;
+import com.netflix.spinnaker.kork.jedis.RedisClientDelegate;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+/** Cache for Helm OCI Docker Registry data. */
+@Service
+@Qualifier("HelmOciDockerRegistryCache")
+public class HelmOciDockerRegistryCache extends DockerRegistryCache {
+
+  static final String ID = "helmOciDockerRegistry";
+
+  // docker-digest must conform to hash:hashvalue. The string "~" explicitly avoids this to act as
+  // an "empty" placeholder.
+  private static final String EMPTY_DIGEST = "~";
```

Comment:
Eh? Why can't we just use null or empty string? Os it `~` special to docker? If not, I would like to avoid magic values where possible